data/README.md

```diff
@@ -123,10 +123,10 @@ data
 
 ## Compilation process
 
-The data compilation is made of indiviual processing steps, where each step adds new or modifies the current data. The
-basic structure of the data however stays the same from the beginning on and is specified in `data-format_*.yaml`.
+The data compilation is made of individual processing steps, where each step adds new or modifies the current data. The
+basic structure of the data, however, stays the same from the beginning on and is specified in `data-format_*.yaml`.
```

Comment:
PSA: "However" cannot live in the middle of a sentence, only the first word.

I have also added a bit more context below

```suggestion
The data compilation is made of individual processing steps, where each step adds new or modifies the current data.
The final structure of the data, is specified in `data-format_*.yaml`.
Some work is underway to ensure that this format is actually being followed via simplifying the data backend and migrating the database server from managing a json blob to "real" tables. This is not done yet.
```

Reply:
<img width="802" alt="image" src="https://github.com/user-attachments/assets/986ac20c-ce1e-4137-ae3b-2f150eb1dedb">
;)

Reply:
hmm...
Okay, fair 😅